server.js

```diff
@@ -42,6 +42,11 @@ const freeformServer = require('./question-servers/freeform.js');
 const cache = require('./lib/cache');
 const { LocalCache } = require('./lib/local-cache');
 const workers = require('./lib/workers');
+const {
+    cleanupMountDirectories,
+    updateExecutorImageTag,
+    ensureExecutorImage,
+} = require('./lib/code-caller-docker');
```

Comment:
In cases where we are only going to call these functions once or twice each, I personally find it easier to read when we import the library itself and then call the functions as `codeCallerDocker.cleanupMountDirectories()`. Otherwise when I'm reading the init code it's not obvious where `cleanupMountDirectories` comes from.

I do agree that sometimes it's nicer to import the functions directly, such as when we are going to call them many times and it's annoying to always have to put `codeCallerDocker.` in front of each call.

Reply:
Resolved via the refactor to `codeCallerDocker.init()` in 695dfd3